source/js/buyers-guide/components/filter/filter.scss

```diff
@@ -26,16 +22,16 @@
   }
 
   .filter-label {
-    display: none;
     position: absolute;
-    right: 2*$filter-bubble-dimension - 1.5em;
-    top: calc(#{$filter-bubble-dimension} / 2 + 2px);
+    top: 2.2em;
+    right: 6.3em;
     color: #666;
     font-size: 12px;
     font-weight: 700;
 
-    @media screen and (min-width: 550px) {
-      display: initial;
+    @media (min-width: $bp-md) {
+      top: calc(#{$filter-bubble-dimension} / 2 + 2px);
+      right: calc(2 * #{$filter-bubble-dimension} - 1.5em);
```

Comment:
Using interpolation `#{}` makes sure `calc()` works for mixed units calculations.